### PR TITLE
docs: update references to NetworkConfiguration::extra_sets

### PR DESCRIPTION
**Pull request by altaua**
Since paritytech/substrate#14080, this struct field no longer exists, now the `add_notification_protocol()` function of
`sc_network::config::FullNetworkConfiguration` is used.

Also neuter the doc links for now; rustdoc can't resolve them (presumably because sc_network::config isn't in scope, though weirdly enough even spelling the link out as
``[`FullNetworkConfiguration`](struct@sc_network::config::FullNetworkConfiguration)`` doesn't work?). Normally this wouldn't be an issue and rustdoc would just not generate links, but rust 1.70 has a bug that completely crashes rustdoc in this case.

Required for https://github.com/paritytech/ci_cd/issues/816

### CODE REVIEW

File: node/network/bridge/src/lib.rs [hunk 1]
```diff
@@ -34,7 +34,7 @@ use polkadot_node_network_protocol::{
 
 /// Peer set info for network initialization.
 ///
-/// To be added to [`NetworkConfiguration::extra_sets`].
+/// To be added to [`FullNetworkConfiguration`]().
```
Comment (bkchr) [inline node/network/bridge/src/lib.rs:37]:
```suggestion
/// To be passed to [`FullNetworkConfiguration::add_notification_protocol`](sc_network::config::FullNetworkConfiguration::add_notification_protocol).
```

File: node/network/bridge/src/rx/mod.rs [hunk 2]
```diff
@@ -49,7 +49,7 @@ use polkadot_primitives::{AuthorityDiscoveryId, BlockNumber, Hash, ValidatorInde
 
 /// Peer set info for network initialization.
 ///
-/// To be added to [`NetworkConfiguration::extra_sets`].
+/// To be added to [`FullNetworkConfiguration`]().
```
Comment (bkchr) [inline node/network/bridge/src/rx/mod.rs:52]:
```suggestion
/// To be passed to [`FullNetworkConfiguration::add_notification_protocol`](sc_network::config::FullNetworkConfiguration::add_notification_protocol).
```

File: node/network/bridge/src/tx/mod.rs [hunk 3]
```diff
@@ -31,7 +31,7 @@ use polkadot_node_subsystem::{
 
 /// Peer set info for network initialization.
 ///
-/// To be added to [`NetworkConfiguration::extra_sets`].
+/// To be added to [`FullNetworkConfiguration`]().
```
Comment (bkchr) [inline node/network/bridge/src/tx/mod.rs:34]:
```suggestion
/// To be passed to [`FullNetworkConfiguration::add_notification_protocol`](sc_network::config::FullNetworkConfiguration::add_notification_protocol).
```

**Review by bkchr — APPROVED** [review]

File: node/network/bridge/src/lib.rs (hunk 1 above)
Comment (altaua) [reply node/network/bridge/src/lib.rs]: I've applied your suggestions. I'm still leaving the () link part at the end empty though; rustdoc doesn't seem to be able to resolve `sc_network::config::FullNetworkConfiguration::add_notification_protocol`, and that's triggering the aforementioned crashing bug in 1.70.

**Review by altonen — APPROVED** [review]

File: node/network/bridge/src/lib.rs (hunk 1 above)
Comment (bkchr) [reply node/network/bridge/src/lib.rs]: Then this should be reported upstream.

**Pull request merged**